### PR TITLE
prometheus-process-exporter: init at 0.5.0

### PR DESCRIPTION
**Pull request by 1000101**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @mmahut

### CODE REVIEW

**Comment by mmahut** [comment]
@GrahamcOfBorg build prometheus-process-exporter

File: pkgs/servers/monitoring/prometheus/process-exporter.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  pname = "process-exporter";
+  version = "0.5.0";
+
+  goPackagePath = "github.com/ncabatoff/process-exporter";
+
+  goDeps = ./process-exporter_deps.nix;
+
+  src = fetchFromGitHub {
+    owner = "ncabatoff";
+    repo = "process-exporter";
```
Comment (mmahut) [inline pkgs/servers/monitoring/prometheus/process-exporter.nix]:
```suggestion
    repo = pname;
```

File: pkgs/servers/monitoring/prometheus/process-exporter.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    rev = "v${version}";
+    sha256 = "129vqry3l8waxcyvx83wg0dvh3qg4pr3rl5fw7vmhgdzygbaq3bq";
+  };
+
+   postPatch = ''
+    substituteInPlace proc/read_test.go --replace /bin/cat cat
+    '';
```
Comment (mmahut) [inline pkgs/servers/monitoring/prometheus/process-exporter.nix]:
```suggestion
  '';
```

File: pkgs/servers/monitoring/prometheus/process-exporter.nix (hunk 1 above)
Comment (1000101) [reply pkgs/servers/monitoring/prometheus/process-exporter.nix]: Thanks, fixed.

**Comment by mmahut** [comment]
@GrahamcOfBorg build prometheus-process-exporter

**Comment by mmahut** [comment]
@GrahamcOfBorg build prometheus-process-exporter

**Pull request merged**